gst/edge/edge_sink.c

```diff
@@ -265,6 +265,16 @@ gst_edgesink_finalize (GObject * object)
     self->host = NULL;
   }
 
+  if (self->dest_host) {
+    g_free (self->dest_host);
+    self->dest_host = NULL;
+  }
+
+  if (self->topic) {
+    g_free (self->topic);
+    self->topic = NULL;
+  }
+
```

Comment:
```suggestion
  g_free (self->dest_host);
  self->dest_host = NULL;
  g_free (self->topic);
  self->topic = NULL;
```

https://developer-old.gnome.org/glib/stable/glib-Memory-Allocation.html#g-free

Reply:
Thanks for your comment!
I know, if mem is Null, it simply returns, so there is no need to check mem against before calling this function.
But i want to unify exception condition other codes in this function.
I think, code uniformity can be an important factor in readability.

So i want to maintain current patch. 
Please advise if the modification is necessary.

Thanks :)

Reply:
@linuxias Then, it is better to fix the other lines for consistency.

Reply:
@wooksong I fixed all lines what free memory in sink and source finalize method.
Please review! Thanks!